hedyorg/hedy
# [RU] student list improvements

RU / New Media Lab issue

[Trello issue](https://trello.com/c/EUgENLGP/24-feature-ui-improve-student-list-bubbles)

**Description**
Improves the student list card. 
* Bubbles represent adventures / quiz. 
* Gray bubble indicates an adventure has not been attempted, blue means it has been attempted (regardless of whether it was a successful run or not), green bubble means the quiz has been finished. Blue border means adventure in progress, green border means quiz in progress. 
* On hover you should see the corresponding adventure.
* If multiple levels are selected, the student list becomes horizontally scrollable. The student name column stays sticky on the left side. 

**Relevant technical choices**
-

***Test instructions***
* Check whether all functionalities still work (hiding/showing sections, opening student details, pop-up, resolving students, etc. 
* Select one level and see the student list.
  * Check whether you think it looks visually appealing.
  * See hovering on bubbles shows the correct corresponding adventures names.
  * See the information is accurate and corresponds to the class overview.
* Select multiple levels and check the same things as for one level.
  * See you can scroll horizontally (>=3 levels selected), see the student names column stays sticky on the right.
